.github/workflows/tests.yml

```diff
@@ -208,3 +208,11 @@ jobs:
           CI: "false" # we pretend to run locally - even if tests fail on the CI, they count towards test coverage
           CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
           DBMS: "postgresql"
+  # This is https://github.com/marketplace/actions/gradle-wrapper-validation
+  # It ensures that the jar file is from gradle and not by a strange third party.
+  gradlevalidation:
```

Comment:
It should run before any gradle task otherwise it's nonsene

Reply:
They run in parallel as you see below...

Reply:
Maybe it's even easier to just include the step before running any Gradle actions,  i.e. first step after checkout `uses: gradle/wrapper-validation-action@v1`. Dunno how long the task takes, but in taht case we are sure that the test fails which will be valued a lot more than just a parallel task that can be ignored?

But it's ok like that as well.

Reply:
It's unlikely that we get get a malicious file...